include/lsst/afw/cameraGeom/Detector.h

```diff
@@ -89,21 +90,23 @@ class Detector final : public table::io::PersistableFacade<Detector>, public tab
              lsst::geom::Box2I const &bbox, lsst::afw::table::AmpInfoCatalog const &ampInfoCatalog,
              Orientation const &orientation, lsst::geom::Extent2D const &pixelSize,
              TransformMap::Transforms const &transforms,
-             CrosstalkMatrix const &crosstalk = CrosstalkMatrix());
+             CrosstalkMatrix const &crosstalk = CrosstalkMatrix(),
+             std::string const &physicalType="unknown");
```

Comment:
What do you think about just using an empty string to mean "unknown"?  It's an easier sentinel value to remember.

Reply:
Should be OK in C++; yamlCamera can still assign "unknown" which I prefer in the end